app/models/concerns/applicable_to_nations.rb

```diff
@@ -0,0 +1,30 @@
+module ApplicableToNations
+  extend ActiveSupport::Concern
+
+  included do
+    validate :applicable_to_at_least_one_nation
```

Comment:
Added a validation to check that it's a valid URL. Didn't use the example that you linked to because it's not actually used anywhere in Whitehall.